test/phoenix_digest_test.exs

```diff
@@ -31,10 +35,21 @@ defmodule BootlegPhoenix.PhoenixDigestTest do
     end)
 
     Enum.each(["deps.get", "bootleg.build", "bootleg.deploy", "bootleg.start"], fn cmd ->
+      Logger.info "-> running mix #{cmd}.."
```

Comment:
This should not be here. Consider refactoring the `BootlegPhoenix.FunctionalCase` to provide a wrapper for `System.cmd("mix", [cmd], [env: shell_env, cd: location])` that includes logging to `debug`.